packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -329,10 +329,16 @@ function VM-Install-From-Zip {
         [Parameter(Mandatory=$false)]
         [string] $executableName, # Executable name, needed if different from "$toolName.exe"
         [Parameter(Mandatory=$false)]
-        [switch] $withoutBinFile # Tool should not be installed as a bin file
+        [switch] $withoutBinFile, # Tool should not be installed as a bin file
+        [Parameter(Mandatory=$false)]
+        [string] $unzipLocation
     )
     try {
-        $toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
+        if ($unzipLocation) {
+            $toolDir = Join-Path $unzipLocation $toolName
```

Comment:
In the case of `psnotify` the folder has the toolName. But I wonder if it would be better to leave this more generic in case another tool requires a path with a different name. 🤔

Reply:
I think this comment still applies, or am I missing something?

Reply:
I am inclined to leave it as this for the moment, because this is how we treat other tools already (each tool is unzipped into a default directory + the tool name), so this change only adds the ability to change the default directory.

If we require modifying the name of the directory that the tool gets unzipped into as well, then I think that we can consider that when the case arises. :)